### DISCUSSION TITLE
crew-launcher root

### DISCUSSION THREAD

**Discussion by eudy97** [skycocker/chromebrew#6933] [Q&A]
How do I create crew-launcher app shortcuts with sudo privileges, specifically nautilus? 

So far I've been using sudo nautilus but it'll be nice to be able to do it from the app shortcut.

**Comment by uberhacker** [comment]
@eudy97: Under no circumstances should you be using sudo to launch apps.  All apps should run as the normal chronos user.  Why do you need sudo to launch nautilus?

**Reply by eudy97** [reply]
> @eudy97: Under no circumstances should you be using sudo to launch apps. All apps should run as the normal chronos user. Why do you need sudo to launch nautilus?

I can't move folders around the file system unless I'm root

**Reply by eudy97** [reply]
specifically /usr/local/

**Reply by satmandu** [reply]
FYI `nautilus` and all of Chromebrew lives in `/usr/local`, so if you move that with nautilus, expect things to break badly.

**Reply by eudy97** [reply]
> FYI `nautilus` and all of Chromebrew lives in `/usr/local`, so if you move that with nautilus, expect things to break badly.

I'm moving stuff to /usr/local/

**Reply by uberhacker** [reply]
> > FYI `nautilus` and all of Chromebrew lives in `/usr/local`, so if you move that with nautilus, expect things to break badly.
> 
> I'm moving stuff to /usr/local/

That could end up breaking things badly too.  What do you need to move and why?

**Reply by eudy97** [reply]
> > > FYI `nautilus` and all of Chromebrew lives in `/usr/local`, so if you move that with nautilus, expect things to break badly.
> > 
> > 
> > I'm moving stuff to /usr/local/
> 
> That could end up breaking things badly too. What do you need to move and why?

where do you recommend I run my programs from? I've made a habit of doing it from there.

I have Discord, Flatpak from source (flatpak installed using crew doesn't work) and Lutris stored there.
